package.nls.json

```diff
@@ -100,7 +101,8 @@
     "AWS.command.ecr.deleteTag": "Delete Tag...",
     "AWS.ecs.enableEcsExec": "Enable Command Execution",
     "AWS.ecs.disableEcsExec": "Disable Command Execution",
-    "AWS.ecs.runCommandInContainer": "Run command in container",
+    "AWS.ecs.runCommandInContainer": "Run Command in Container",
+    "AWS.ecs.openTaskInTerminal": "Open Task in Terminal",
```

Comment:
I strongly suspect that "Open Terminal" will be more intuitive than "Open Task in Terminal". As a developer, the latter sounds like a new/unfamiliar feature, unrelated to the intuitive idea of "I want a terminal here". People viewing this list aren't necessarily going to remember which node is a "container" or a "task" or "...". But they do know intuitively that right-clicking and choosing "Open Terminal" will probably put a terminal on whatever they right-clicked.

Reply:
Isn't `Run Command in Container` operating on the same kind of object? There also I'd suggest we rename it to `Run Command...`